apps/src/code-studio/pd/foorm/editor/components/FoormEntityEditorHeader.jsx

```diff
@@ -1,21 +1,15 @@
-// Metadata and buttons for the Foorm Editor. Shows form name and version,
+// Metadata and buttons for the Foorm Editor. Shows name and version,
```

Comment:
this would be better as a multiline comment above the class declaration

Reply:
Moved all multiline comments at the top of these files to above the class declaration.